programs/whirlpool/src/instructions/open_position_with_metadata.rs

```diff
@@ -32,7 +32,7 @@ pub struct OpenPositionWithMetadata<'info> {
     pub position_mint: Account<'info, Mint>,
 
     /// CHECK: checked via the Metadata CPI call
-    /// https://github.com/metaplex-foundation/metaplex-program-library/blob/master/token-metadata/program/src/utils.rs#L873
+    /// https://github.com/metaplex-foundation/mpl-token-metadata/blob/2f93406c62e791ee93435644c1a3507e910a116b/programs/token-metadata/program/src/utils/metadata.rs#L78C5-L78C39
```

Comment:
Maybe leave this one referring to master (since they can update their program whenever they want)